chainercv/utils/testing/parameterized.py

```diff
@@ -0,0 +1,23 @@
+import six
+
+from chainer import testing
+
+
+def parameterize(*params):
```

Comment:
Can you add comment on why this method is needed?

Reply:
updated